ted/src/main/java/frc/robot/subsystems/CameraSubsystem.java

```diff
@@ -3,51 +3,118 @@
 // Home of the Cybears
 // FRC - Crescendo - 2024
 // File: CameraSubsystem.java
-// Intent: Forms a stub for the prelminary named subsystem above.
+// Intent: Forms the prelminary code for camera train subsystem.
 // ************************************************************
 
 // ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import frc.robot.common.VisionMeasurement;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import frc.robot.common.DistanceMeasurement;
 
+/**
+ * A class to encapsulate the camera subsystem
+ */
 public class CameraSubsystem extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
-  public CameraSubsystem() {}
+  private final double milisecondsInSeconds = 1000.0;
+  private final int defaultDoubleArraySize = 7;
+  private final int TimestampIndex = 6;
+  private final int botPositionXIndex = 0;
+  private final int botPositionYIndex = 2;
+  private final int botRotationIndex = 5;
+  private final int noTagInSightId = -1;
+  NetworkTable table = NetworkTableInstance.getDefault().getTable("limelight");
```

Comment:
Why do we need a public NetworkTable member?  If you don't need it exposed, please make it private

Reply:
made private